lib/node_modules/@stdlib/repl/lib/keybindings.js

```diff
@@ -0,0 +1,347 @@
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Table mapping of REPL actions and their corresponding keybinding triggers.
+*
+* @name KEYBINDINGS
+* @type {Object}
+*/
+var KEYBINDINGS = {
+	'moveRight': [
+		{
+			'name': 'right',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'f',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'moveLeft': [
+		{
+			'name': 'left',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'b',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'moveWordRight': [
+		{
+			'name': 'right',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'f',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'moveWordLeft': [
+		{
+			'name': 'left',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'b',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'moveBeginning': [
+		{
+			'name': 'home',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'a',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'moveEnd': [
+		{
+			'name': 'end',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'e',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'tab': [
+		{
+			'name': 't',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'indentLineRight': [
+		{
+			'name': 'right',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		},
+		{
+			'name': 'i',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'indentLineLeft': [
+		{
+			'name': 'left',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		},
+		{
+			'name': 'i',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'deleteLeft': [
+		{
+			'name': 'backspace',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'h',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'deleteRight': [
+		{
+			'name': 'delete',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'd',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'deleteWordLeft': [
+		{
+			'name': 'backspace',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'backspace',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		},
+		{
+			'name': 'w',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
```

Comment:
Should all of these keybindings work or just one of them?

Reply:
All of them should work. The user can set multiple keybindings to trigger a single action.

Reply:
It was often the case that only one of the keybindings worked. I suppose it is possible that there are conflicts happening somewhere so that not able to see intended changes.